browser.js

```diff
@@ -307,9 +307,11 @@ function noop () {}
 /* eslint-disable */
 /* istanbul ignore next */
 function pfGlobalThisOrFallback () {
-  const defd = (o) => typeof o !== 'undefined' && o
+  var defd = function (o) {
+    return typeof o !== 'undefined' && o;
```

Comment:
this will fail test (which includes linting)

Reply:
(stray ;)

Reply:
Fixed.